project.clj

```diff
@@ -0,0 +1,9 @@
+(defproject jannis/groom "0.1.0-SNAPSHOT"
+  :dependencies [[camel-snake-kebab "0.4.0"]
+                 [org.clojure/clojure "1.9.0"]
+                 [org.clojure/clojurescript "1.10.238"]
+                 [org.clojure/spec.alpha "0.1.143"]
+                 [org.clojure/test.check "0.10.0-alpha2"]
+                 [sablono "0.8.3"]]
+  :source-paths ["src/main"]
+  :test-paths ["src/test"])
```

Comment:
We can remove the project.clj if you want, I just wanted to get a REPL going and wasn't sure what the plan was regarding boot or deps.edn.